.github/workflows/build-lint-test.yml

```diff
@@ -24,11 +24,19 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v4
         with:
-          node-version: 20
+          node-version: 22
           cache: pnpm
 
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+
       - name: Install dependencies
         run: pnpm install
 
+      - name: create test env file
+        run: |
+          touch packages/sdk/.env
```

Comment:
Should be able to do without this, I think?

```suggestion
```

Reply:
yep you're right!